.gitignore

```diff
@@ -135,6 +135,7 @@ dynamodb_local_latest/
 # client (Vite)
 client/dist
 client/.env
+client/.vite
```

Comment:
I don't know what I did or if I forgot this in the initial PR but I noticed after running the formatting that this folder was added, so I removed it and added it to the gitignore - lmk if I should do that in a different PR though!